containers/docker-existing-dockerfile/.devcontainer/devcontainer.json

```diff
@@ -1,22 +1,26 @@
 {
-    // "See https://aka.ms/vscode-remote/devcontainer.json for format details."
-    "name": "Existing Dockerfile",
+	// See https://aka.ms/vscode-remote/devcontainer.json for format details.
```

Comment:
We should probably update the automatically added comment to this and we need to make sure we preserve the comments when applying a definition. (I can do that.)